src/app/layout.tsx

```diff
@@ -17,12 +18,15 @@ export async function generateMetadata(): Promise<Metadata> {
 export default async function RootLayout({ children }: { children: ReactNode }) {
   const headersList = headers();
   const tokenPrice = await getTokenPrice();
```

Comment:
Seems like tokenPrice and statusBarContent can be moved further down into the components that directly need the data.

Reply:
We're doing it here because want to run them on server:
1) token price is caches on server
2) status bar request on client would expose API key

Reply:
I don't see server side props anywhere, so curious to know how exactly is this running on the server and not the client?